src/nimblepkg/packageinfo.nim

```diff
@@ -18,6 +18,21 @@ proc initPackageInfo*(): PackageInfo =
 proc initPackage*(): Package =
   result = Package(version: notSetVersion)
 
+proc `$`*(meth: DonationMethod): string {.inline.} =
+  ## Turns a `DonationMethod` into a pretty string.
+  case meth
+  of DonationMethod.GitHub: "GitHub"
+  of DonationMethod.OpenCollective: "OpenCollective"
+  of DonationMethod.Patreon: "Patreon"
```

Comment:
I think this should be just a string metadata instead of just 3 hardcoded "provider companies",
some people prefer to use USDT/BTC/ETH/crypto/other services etc, 
and in some places of the world the hardcoded services are not available (as donations collector).

Reply:
Sounds alright, but we'd also need to properly audit any new packages that are added in. We need to ensure that none of them end up linking to a phishing site, albeit the odds are very low.

Reply:
Thats what I tried to do at
https://github.com/juancarlospaco/nim_packages_security_audit?tab=readme-ov-file#fully-automated-nim-packages-security-audit
Feel free to contribute/revive, maybe it needs like a "cool web visualization" (?). 

Also the package scanner can be improved too:
https://github.com/nim-lang/packages/blob/master/package_scanner.nim

I think the efforts there are so much better spent.

Reply:
This might sound stupid - but what about using something like URL Haus, VirusTotal or Google Safebrowsing to the package scanner?

Reply:
Nevermind, URL Haus does not keep track of phishing URLs (my main concern).

Reply:
@juancarlospaco I'd say we should leave the URL auditing to the person who's going to merge the package into the index. Most browsers have SafeBrowsing built in either ways. Do you have any other suggestions?

Reply:
Enforce a common-sense standard max len for the string ? 256 char or something here or in the package scanner. IMHO 🤔